pkg/logs/logs.go

```diff
@@ -62,22 +59,20 @@ func (writer GlogWriter) Write(data []byte) (n int, err error) {
 
 // InitLogs initializes logs the way we want for kubernetes.
 func InitLogs(fs *flag.FlagSet) {
+	logs.InitLogs()
+
 	if fs == nil {
 		fs = flag.CommandLine
 	}
-	klog.InitFlags(fs)
 	_ = fs.Set("logtostderr", "true")
```

Comment:
Is this flag still present? Is the new default still to not log to stderr? Can we set this setting somewhere else?

Reply:
yes, it is still present with default `true` https://github.com/kubernetes/klog/blob/d7fc50528570f8276de46570f916682257821e22/klog.go#L407-L408

Reply:
@inteon do you think we should move it to `options.go`?

Reply:
Let's keep it here for now. We will deprecate this flag as part of #5879.